requirements-tests.txt

```diff
@@ -6,7 +6,7 @@ flake8-noqa==1.3.0        # must match .pre-commit-config.yaml
 flake8-pyi==22.11.0       # must match .pre-commit-config.yaml
 isort==5.11.3             # must match .pre-commit-config.yaml
 mypy==0.991
-packaging==21.3
+packaging==22.0
```

Comment:
This needs to be bumped, as `packaging.requirements.Requirement` doesn't have a stable hash on `<22.0` (https://github.com/pypa/packaging/commit/aebc072a06925cc0004b031e6b6f3028e5e2e686)

Reply:
Actually, this isn't needed, I'm confusing myself. Might be good to do it anyway, but not necessary for this PR.